src/Services/ReorderElements.php

```diff
@@ -92,24 +93,34 @@ public function reorder($elementToBeAfterID = 0)
 
         // We are updating records with SQL queries to avoid the ORM triggering the creation of new versions
         // for each element that is affected by this reordering.
-        $tableName = Convert::raw2sql(DataObject::getSchema()->tableName(BaseElement::class));
-
-        if ($sortAfterPosition < $currentPosition) {
-            $operator = '+';
-            $filter = "\"$tableName\".\"Sort\" > $sortAfterPosition AND \"$tableName\".\"Sort\" < $currentPosition";
-            $newBlockPosition = $sortAfterPosition + 1;
-        } else {
-            $operator = '-';
-            $filter = "\"$tableName\".\"Sort\" <= $sortAfterPosition AND \"$tableName\".\"Sort\" > $currentPosition";
-            $newBlockPosition = $sortAfterPosition;
+        $baseTableName = Convert::raw2sql(DataObject::getSchema()->tableName(BaseElement::class));
+
+        // Update both the draft and live versions of the records
+        $suffixes = [''];
+        if (BaseElement::has_extension(Versioned::class)) {
+            $suffixes[] = '_Live';
```

Comment:
For future proofing, it might be better to use `Versioned::stageTable()` to determine the table name rather than hardcoding the suffix.